examples/pytorch/graphsage/advanced/train_sampling_unsupervised.py

```diff
@@ -73,7 +73,7 @@ def run(proc_id, n_gpus, args, devices, data):
     if args.data_device == 'gpu':
         nfeat = nfeat.to(device)
     elif args.data_device == 'uva':
-        nfeat = dgl.contrib.UnifiedTensor(nfeat, device=device)
+        nfeat = dgl.utils.pin_memory_inplace(nfeat)
```

Comment:
`dgl.utils.pin_memory_inplace()` returns a `ndarray` handle that holds the pinning flag. So I'd suggest either the following:
1. 
```python
nfeat_nd = dgl.utils.pin_memory_inplace(nfeat)
```
and just use `nfeat` in the following code.
2. Use PyTorch `.pin_memory()`
```python
nfeat = nfeat.pin_memory()
```
3. Use PyTorch's in-place pinning.
```python
torch.cuda.cudart().cudaHostRegister(nfeat.data_ptr(), nfeat.numel() * nfeat.element_size(), 0)
```
Generally, I prefer 2 due to the simplicity, though it requires more CPU PEAK memory during copying `nfeat` to the pinned buffer.

Reply:
Thanks for explaining. And I agree with you the 2end one is preferable because simplicity and clearness is at high priority in example. Let me fix it using method 2.